internal/olm/installer/manager.go

```diff
@@ -27,7 +27,7 @@ import (
 
 const (
 	DefaultVersion = "latest"
-	DefaultTimeout = time.Minute * 2
+	DefaultTimeout = time.Minute * 3
```

Comment:
While I don't have a really strong opinion on this change, it _does_ have the downside that if something goes wrong with the installation, users will have to wait an extra minute to find out.

These commands have a `--timeout` flag that overrides the default timeout for situations where it takes OLM longer than usual to install.

Reply:
```
$ build/operator-sdk olm install --help
Install Operator Lifecycle Manager in your cluster

Usage:
  operator-sdk olm install [flags]

Flags:
  -h, --help               help for install
      --timeout duration   time to wait for the command to complete before failing (default 2m0s)
      --version string     version of OLM resources to install (default "latest")

Global Flags:
      --verbose   Enable verbose logging
```

I'm not sure changing the timeout in the code makes sense since we do offer a `--timeout`.  On one cluster it might be 3 minutes. On another it might need to be as high as 5 minutes. Now if the default OLM install on OCP always takes longer than 2 minutes then it might make sense to set a better default.

Reply:
@jianzhangbjz What OCP have we tested it against? AWS? OpenStack? etc?

Reply:
@joelanford Yeah, thanks! I saw that `--timeout` flag. But, for the beginner users, they don't know it will solve this `context deadline exceeded` error. And, I tried it multi times on OCP 4.6 running on AWS, all failed by using the default timeout. So, I guess it's better to change it to `3 mins`.

Reply:
@jmrodri only test OCP 4.6 on AWS.

Reply:
@jianzhangbjz Forgive my ignorance, but how is the `olm install` command tested on OCP, where OLM is already present? Is it possible to manually remove OLM and tell CVO not to re-install it?

While we don't have any specific platform requirements, we generally expect users to be using the `install` and `uninstall` commands on non-OCP clusters that don't have OLM installed and managed by default.

Also, can you tell which part of the process is taking the longest?

Reply:
@joelanford You're right! Yes, I stopped the CVO first, like this `oc scale --replicas 0 -n openshift-cluster-version deployments/cluster-version-operator`. And, remove the OLM manually.
> While we don't have any specific platform requirements, we generally expect users to be using the install and uninstall commands on non-OCP clusters that don't have OLM installed and managed by default.

Ok, got it, thanks! Which is the best way to create a cluster without the OLM installed as default?
> Also, can you tell which part of the process is taking the longest?

Manual removing the OLM took much time. And for the install process, I think fetching the OLM version, resource took a lot of time.

Reply:
@jianzhangbjz when we create clusters without OLM, we simply use a kind cluster ( https://kind.sigs.k8s.io/docs/user/quick-start/) or minikube ( https://kubernetes.io/docs/tasks/tools/install-minikube/) .

I like to use kind and the process looks like this:

```
kind delete cluster
kind create cluster
operator-sdk olm install
```